migration/migrating-3-4/deploying_cam.adoc

```diff
@@ -0,0 +1,27 @@
+[id='deploying-cam_{context}']
+= Deploying the Cluster Application Migration tool
+include::modules/common-attributes.adoc[]
+:context: migrating-3-4
+:migrating-3-4:
+
+toc::[]
+
+Deploying the CAM tool involves the following steps:
+
+. Installing the CAM Operator on the xref:installing-cam-operator-ocp-3_sourcecluster-3-4[source] and xref:installing-cam-operator-ocp-4_targetcluster-3-4[target] clusters
```

Comment:
A bit confusing that these are numbered steps, but the corresponding steps aren't numbered. Also, could you just remove these "pre-steps" and go right to the procedures? Perhaps you could start with an intro sentence, such as, "Deploying the CAM tool requires installing the CAM Operator and configuring cross-resource sharing."

Reply:
Same bit about the xref requiring full path.